docs/usage.md

```diff
@@ -0,0 +1,60 @@
+# ESI Usage
+
+This document highlights OpenStack CLI commands used to perform common ESI functions. A complete listing of commands can be found by reading the [complete OpenStack CLI documentation](https://docs.openstack.org/python-openstackclient/latest/cli/command-list.html). Each entry here also links to the appropriate entry in the OpenStack CLI documentation.
+
+* [Identity](#identity)
+* [Isolation](#isolation)
+* [Maintenance](#maintenance)
+* [Provisioning](#provisioning)
+
+## <a name="identity"></a>Identity
+
+### Administrator
+
+* **Create project:** `openstack project create <project-name>` [[full reference](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/project.html#project-create)]
+* **Create user:** `openstack user create --project <project-name> --password <password> <user-name>` [[full reference](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/user.html#user-create)]
+* **Give user role in project:** `openstack role add --project <project-name> --user <user-name> <role>` [[full reference](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/role.html#role-add)]
+  * By default, admins should give users the ``member`` role in a project.
+
+### User
+
+* **Set your password:** `openstack user password set`
+
```

Comment:
Should we include a complete example here of creating a project, user, and performing role assignment?

Reply:
Yep - I think that's right above, in the "Administrator" section.

Reply:
...not it's not?  I mean, we document the commands, but we don't provide a complete example.

Reply:
Ah, gotcha. Should be added now!